airbyte-webapp/packages/eslint-plugin/no-hardcoded-connector-ids.js

```diff
@@ -2,14 +2,14 @@ const destinations = require("../../src/utils/connectors/destinations.json");
 const sources = require("../../src/utils/connectors/sources.json");
 
 // Create a map for connector id to variable name (i.e. flip the direction of the map)
-const sourceIds = Object.fromEntries(Object.entries(sources).map((entry) => [entry[1], entry[0]]));
-const destinationIds = Object.fromEntries(Object.entries(destinations).map((entry) => [entry[1], entry[0]]));
+const sourceIdToName = Object.fromEntries(Object.entries(sources).map((entry) => [entry[1], entry[0]]));
+const destinationIdToName = Object.fromEntries(Object.entries(destinations).map((entry) => [entry[1], entry[0]]));
 
 const validateStringContent = (context, node, string) => {
```

Comment:
Nit: Can we call this something other than string?

Reply:
Happy for any suggestions :D Couldn't come up with anything better, since this is literally just the content of any `string` in the UI, we don't have any more knowledge what this is or isn't.

Reply:
Maybe `nodeContent`?